lib/smart_proxy_remote_execution_ssh/command_action.rb

```diff
@@ -42,7 +42,7 @@ def rescue_strategy_for_self
     def command
       @command ||= Dispatcher::Command.new(:id               => input[:task_id],
                                            :host             => input[:hostname],
-                                           :ssh_user         => 'root',
+                                           :ssh_user         => input[:ssh_user] || 'root',
```

Comment:
I'd actually be more inclined to remove the || case, and make foreman always send the relevant user name. Any reason to configure it on the proxy side?